docs/gmi.md

```diff
@@ -163,6 +163,9 @@ The game should use this flag to decide whether to show the exit button
 full screen status itself. This flag will be false if the game is currently
 embedded in an iframe or true if the page has been redirected to the Playpen.
 
+### gmi.isDebugMode
+A boolean that indicates if the game should be in debug mode or not. If true, all levels should be unlocked for testing purposes.
```

Comment:
I don't think we need to say it unlocks all levels as that will change from game to game? but we can say it provides a mechanism for people to enable a debug mode via external config.